pkg/scan/scan.go

```diff
@@ -15,6 +15,24 @@ import (
 	"github.com/dropalldatabases/sif/pkg/logger"
 )
 
+func fetchRobotsTXT(url string, client *http.Client) (*http.Response, error) {
+	resp, err := client.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode == http.StatusMovedPermanently {
+		redirectURL := resp.Header.Get("Location")
+		if redirectURL == "" {
+			return nil, fmt.Errorf("redirect location is empty")
+		}
```

Comment:
please look into the way we handle logging. do not return an error and use our log package